compiler/one-cmds/one-import-onnx

```diff
@@ -61,6 +61,14 @@ def _get_parser():
     tf2tfliteV2_group.add_argument('--model_format', default='saved_model')
     tf2tfliteV2_group.add_argument('--converter_version', default='v2')
 
+    # save intermediate file(s)
+    parser.add_argument(
+        '--save_intermediate',
+        action='store_const',
+        dest='save_intermediate',
+        const='--save_intermediate',
+        help='Save intermediate files to output folder')
```

Comment:
If it just stores boolean value, then it can be like below.
```cxx
converter_version.add_argument(
        "--save_intermediate", action="store_true", help="Save intermediate files to output folder")
```

Reply:
Ah... got this error
```
TypeError: __init__() missing 1 required positional argument: 'const'
```

Reply:
@seanshpark Apparently,, that error occurs when `action='store_const'`. Could you try one more time with `action='store_true'`?

Reply:
This works :)